doc/api/cli.md

```diff
@@ -192,7 +192,7 @@ added: v8.5.0
 -->
 
 Enable latest experimental modules features (currently
```

Comment:
Can this say that it _allows_ one to enable them instead? This is misleading...

Reply:
It does enable them, it is an alias

Reply:
You're right, disregard. Those flags throw errors when used by themselves, which threw me off.